src/app/static/src/app/store/translations/locales.ts

```diff
@@ -579,6 +585,8 @@ const en: Translations = {
     uniqueProjectName: "Please choose a unique name",
     updatePassword: "Update password",
     upload: "Upload",
+    uploadFromZip: "Uploading project from zip",
```

Comment:
Can we keep the name we use to refer to the file consistent, looks like using `model outputs` elsewhere. So `Uploading project from model outputs`?

Reply:
see above.